python-lib/prologin/concours/settings.py

```diff
@@ -24,6 +24,18 @@
 
 SITE_ID = 1
 
+
+SITE_NAME = cfg['contest']['game']
+
+# False during final event, True to attach to prologin's website
+RUNNING_ONLINE = True
```

Comment:
J'ai déjà dit que j'aimais pas ce nom ? Un truc avec PUBLIC me paraît déjà plus chargé sémantiquement. Là le running apporte rien et online bah c'est un site web quoi. Je comprends avec le contexte que tu veux dire online on prologin.org rather than the internal isolated finals only version, mais c'est pas assez évident.

C'est marrant parce que ton commentaire est clair et précis et contient tous les mots clés importants, et puis juste après "RUNNING_ONLINE" genre quoi.